packages/teams-js/src/public/appInitialization.ts

```diff
@@ -15,86 +15,84 @@ import { app } from './app';
  */
 const appInitializationTelemetryVersionNumber: ApiVersionNumber = ApiVersionNumber.V_1;
 
-export namespace appInitialization {
-  /**
-   * @deprecated
-   * As of TeamsJS v2.0.0, please use {@link app.Messages} instead.
-   */
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  export import Messages = app.Messages;
-  /**
-   * @deprecated
-   * As of TeamsJS v2.0.0, please use {@link app.FailedReason} instead.
-   */
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  export import FailedReason = app.FailedReason;
-  /**
-   * @deprecated
-   * As of TeamsJS v2.0.0, please use {@link app.ExpectedFailureReason} instead.
-   */
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  export import ExpectedFailureReason = app.ExpectedFailureReason;
-  /**
-   * @deprecated
-   * As of TeamsJS v2.0.0, please use {@link app.IFailedRequest} instead.
-   */
-  export import IFailedRequest = app.IFailedRequest;
-  /**
-   * @deprecated
-   * As of TeamsJS v2.0.0, please use {@link app.IExpectedFailureRequest} instead.
-   */
-  export import IExpectedFailureRequest = app.IExpectedFailureRequest;
+/**
+ * @deprecated
+ * As of TeamsJS v2.0.0, please use {@link app.Messages} instead.
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export { Messages };
```

Comment:
I am so happy we are changing this syntax

Reply:
same